src/pages/home-page/HomePage.tsx

```diff
@@ -1,33 +1,67 @@
 import React, { useState } from "react";
 import { useOutletContext } from "react-router-dom";
-import { PokemonsList } from "./PokemonsList";
 import { SearchBar } from "@/design-system/generic-componenets/SearchBar";
+import { GenericDropDown} from "@/design-system/generic-componenets/GenericDropDown";
+import type { Options } from "@/design-system/generic-componenets/GenericDropDown";
+import { usePokemonsData } from "@/hooks/usePokemonsData";
+import type { SortOption } from "@/hooks/usePokemonsData";
+import { PokemonsList } from "./PokemonsList";
 
 export const HomePage: React.FC = () => {
```

Comment:
```suggestion
export const HomePage = () => {
```

using React.FC is redundant and unnecessary.